packages/in_app_purchase/in_app_purchase_storekit/darwin/Classes/FIAPReceiptManager.m

```diff
@@ -43,4 +43,8 @@ - (NSData *)getReceiptData:(NSURL *)url error:(NSError **)error {
   return [NSData dataWithContentsOfURL:url options:NSDataReadingMappedIfSafe error:error];
 }
 
+- (NSURL *)getReceiptURL {
```

Comment:
nit: can be a property

Reply:
this is a method so it can be stubbed out to return nil, or did u mean something else?

Reply:
You can have this in header file: 
```
@property (nonatomic, readonly) receiptURL; 
```
Then in implementation, you can have 
```
- (NSURL *)receiptURL {
...
}
```

Reply:
done